### PR TITLE
fix continual reload

### PR DESCRIPTION
**Pull request by jstefaniuk-d2l**

### CODE REVIEW

File: src/fileViewer.js [hunk 1]
```diff
@@ -40,6 +40,11 @@ var FileViewer = React.createClass({
 		locale: React.PropTypes.string,
 		progressCallback: React.PropTypes.func
 	},
+	shouldComponentUpdate: function(prevProps, prevState) {
```
Comment (CodeBaboon) [inline src/fileViewer.js]: We probably want to redraw if locale changes too

Comment (jstefaniuk-d2l) [inline src/fileViewer.js]: will locale ever change after the first draw?

Comment (CodeBaboon) [inline src/fileViewer.js]: Maybe? Depends how this is being used by whoever uses it.

**Comment by omsmith** [comment]
Is cool, I guess? No which prop was changing that was causing it to reload?

**Comment by jstefaniuk-d2l** [comment]
I think it has something to do with the internationalization wrapper messing with reacts 'should I redraw' function
If I changed the function to check for all the props it would still fix the problem

**Comment by omsmith** [comment]
https://github.com/Brightspace/react-frau-intl/blob/master/src/i18n.js it doesn't appear to be doing all that much.

**Comment by jstefaniuk-d2l** [comment]
It does create a new react class every time fileviewer renders though

**Comment by jstefaniuk-d2l** [comment]
maybe if I move that out of the render function...

**Comment by omsmith** [comment]
I mean, it shouldn't be re-rendering it as far as I'm aware, that's kinda what React is supposed to take car of

**Comment by jstefaniuk-d2l** [comment]
whoops, one sec

**Comment by jstefaniuk-d2l** [comment]
just need to retest this one more time

**Comment by jstefaniuk-d2l** [comment]
Fixed, shippit

**Comment by CodeBaboon** [comment]
Fancy

**Comment by omsmith** [comment]
If this fixes it, it does seem much better to me

**Comment by CodeBaboon** [comment]
Doesn't re-resolve the FileViewerResolved on src change though :(

**Comment by CodeBaboon** [comment]
But it seems to work, not sure I fully grok the react flow here.

**Comment by CodeBaboon** [comment]
Ok I think I understand it now. LGTM

**Pull request merged**